src/ssWeatherManager.lua

```diff
@@ -155,6 +155,10 @@ function ssWeatherManager:readStream(streamId, connection)
 
         table.insert(self.rains, rain)
     end
+
+    -- we need to synch this, we have an error log entry on dedicated server otherwise
+    -- for now i do it here, it's not elegant but at the moment it work
+    ssSnow.appliedSnowDepth = streamReadFloat32(streamId)

```

Comment:
Odd because snow is only done on the server,  not on the client. There might be a better solution...